packages/components/src/components/relative-datepicker/relative-datepicker.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useRef } from 'react';
+import { toMoment, daysFromTodayTo } from '@deriv/shared/utils/date';
+
+const RelativeDatepicker = props => {
```

Comment:
Please use object destruction for props.

Reply:
done